src/cs/Ssh/SshSession.cs

```diff
@@ -472,6 +476,9 @@ await SendMessageAsync(KeyExchangeInitMessage.None, connectCts.Token)
 					this.OnReportProgress(Progress.CompletedProtocolVersionExchange);
 					IsConnected = true;
 				}
+
+				// Implement KeepAlive
+				ImplementKeepAlive(connectCts.Token);
```

Comment:
We shouldn't use `connectCts.Token` here, `connectCts` is disposed right after the session is connected.

Reply:
Will change it to `this.disposeCancellationSource.Token`